<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/run.py", line 107, in main
    result = linter.lint(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 620, in lint
    msgs = self.lint_file(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 692, in lint_file
    messages = self.lint_recipe(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 749, in lint_recipe
    res = self.check_instances[check].run(recipe, fix)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 258, in run
    self.check_deps(recipe.get_deps_dict())
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/check_build_help.py", line 349, in check_deps
    build_file = self.recipe.get(script)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/recipe.py", line 583, in get
    raise KeyError(f"No '{path}' in Recipe {self}") from None
KeyError: "No 'outputs/0/script' in Recipe /tmp/abs_f1of3lhw4f/clone/recipe/"

ERROR conda.cli.main_run:execute(47): `conda run conda-lint /tmp/abs_f1of3lhw4f/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -69,6 +69,7 @@ outputs:
         - flit.build
       requires:
         - pip
+        - m2-grep  # [win]
       commands:
         - pip check
         - flit --version | grep -Fq {{ version }}

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -44,7 +44,9 @@ outputs:
   - name: flit
     version: {{ version }}
     build:
-      script: cd source/ && FLIT_ROOT_INSTALL=1 python -m flit install --deps=none
+      script: set FLIT_ROOT_INSTALL=1  # [win]
+      script: FLIT_ROOT_INSTALL=1  # [not win]
+      script: cd source/ && {{ PYTHON }} -m flit install --deps=none
       entry_points:
         - flit = flit:main
     requirements:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -45,7 +45,7 @@ outputs:
     version: {{ version }}
     build:
       script: set FLIT_ROOT_INSTALL=1  # [win]
-      script: FLIT_ROOT_INSTALL=1  # [not win]
+      script: export FLIT_ROOT_INSTALL=1  # [unix]
       script: cd source/ && {{ PYTHON }} -m flit install --deps=none
       entry_points:
         - flit = flit:main

